aws-s3-transfer-manager/examples/cp.rs

```diff
@@ -151,6 +152,7 @@ async fn do_download(args: Args) -> Result<(), BoxError> {
     let tm_config = aws_s3_transfer_manager::from_env()
         .concurrency(ConcurrencySetting::Explicit(args.concurrency))
         .part_size(PartSize::Target(args.part_size))
+        .app_name(Some(AppName::new("rust_ts_cp").unwrap()))
```

Comment:
What is ts? I think it would be better to just spell it `transfer_manager_sample_cp`.